src/assets/data/map_data/other.ts

```diff
@@ -1,139 +1,141 @@
+import { Location } from "../../../hooks/useMapData";
+import i18n from "../../../i18n";
+import elevator from "./elevator_positions";
 import {
   FIRST_FLOOR_IMG_URL,
   GROUND_FLOOR_IMG_URL,
   MEZZANINE_FLOOR_IMG_URL,
 } from "./floor_images";
-import i18n from "../../../i18n";
-import elevator from "./elevator_positions";
 
-let locations;
-export default locations = [
+const locations: Location[] = [
   {
-    floor: i18n.t("mezzanine_floor"),
+    floor: String(i18n.t("mezzanine_floor")),
```

Comment:
Same thing as `as string`. If the result is `null`, we would be assigning `'null'` as `String(null) -> 'null'`
Resolve it with `i18n.t(....) ?? undefined`